docs/updating.rst

```diff
@@ -1,36 +1,53 @@
-===============================================================
-How to update the Atomic Developer Bundle to the latest version
-===============================================================
+===========================================================
+How to update Atomic Developer Bundle to the latest version
+===========================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
 
-  1. Check if a newer version of the box is available use this command::
+-------------
+Prerequisites
+-------------
 
-      # vagrant box outdated
+If you cannot directly update your Vagrant box, you must perform the following steps
+to remove the previously-installed boxes from your virtual machine.
 
-  2. If a newer version is available, the output should look like this::
+1. If ADB is running, run the `vagrant halt` command.
 
-        # vagrant box outdated
+2. Delete the Vagrant boxes with the `vagrant destroy` command. This action will also
```

Comment:
Ya agreed that these steps looking like we are teaching vagrant here :wink: .

IMO there are two ways how these boxes are managed by developers usually.
- Automatic download , update:
   If we do `vagrant up` on Vagrantfile where box name describe in `config.vm.box` is not present locally, then vagrant will download box from "atlas.hashicorp.com". This will also allow automatic update with `vagrant box update`.
- Download box, add box:
  Second way is to download the box like anywhere from `cloud.centos.org`, then add through `vagrant box add <box_name> <local box url / remote url>` command and specify this `box_name` in Vagrantfile and does "vagrant up". In this case, automatic update is not possible. Then updating will need download new updated box, add that box. If same name is to be used then probably --version is need to be used otherwise user will get error as box name already exists. Otherwise, user need to remove the box and then add updated box.

Hope you will get some insights on how users generally use.